gradle.properties

```diff
@@ -18,7 +18,7 @@ group=org.sonatype.gradle.plugins
 version=3.0.0-SNAPSHOT
 release.useAutomaticVersion=true
 
-nexusPlatformApiVersion=4.0.7-01
+nexusPlatformApiVersion=5.0.0-01
```

Comment:
Upgrading to 5..0.0-01 prints deprecation warnings for usage of the cyclonedx class `Tool`.  Not familiar with cyclonedx so not sure what the recommended class is to use, or timeline for adoption.

See usages [here ](https://github.com/sonatype-nexus-community/scan-gradle-plugin/blob/a4409e452abf9859b5b5958019039e19a52d61dc/src/main/java/org/sonatype/gradle/plugins/scan/ossindex/CycloneDxResponseHandler.java#L150)and [here](https://github.com/sonatype-nexus-community/scan-gradle-plugin/blob/a4409e452abf9859b5b5958019039e19a52d61dc/src/test/java/org/sonatype/gradle/plugins/scan/ossindex/CycloneDxResponseHandlerTest.java#L268).

Reply:
We can aim to support CycloneDX 1.5 instead in another PR.

Reply:
Also, upgrading to `5.1.2-01` seems quite straightforward:
11982125ad17b7793b921dd605cecb90a71c4104

I'd advocate to update to that latest version.

Reply:
> Also, upgrading to `5.1.2-01` seems quite straightforward:
> [1198212](https://github.com/sonatype-nexus-community/scan-gradle-plugin/commit/11982125ad17b7793b921dd605cecb90a71c4104)

Nice!